pennylane/beta/interfaces/autograd.py

```diff
@@ -121,6 +122,8 @@ def get_parameters(self, trainable_only=True):  # pylint: disable=missing-functi
 
     @autograd.extend.primitive
     def _execute(self, params, device):
+        # unwrap all NumPy scalar arrays to Python literals
+        params = [p.item() if p.shape == tuple() else p for p in params]

```

Comment:
@trbromley, this fixes the bug you noticed with the SF plugin (this bug did not affect built-in devices, hence was not picked up in the tests)